### PR TITLE
Add nemlig.com as Keda users

### PR DESCRIPTION
**Pull request by larssb**
Signed-off-by: Lars Bing Bong <bingchonglars@gdupmail.slum>

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Can you fix the DCO issues please? You can learn more in our [contribution guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what).

File: config.toml [hunk 1]
```diff
@@ -437,6 +437,10 @@ logo = "metamaze.png"
 url = "https://microsoft.com"
 logo = "microsoft.png"
 
+[[params.users]]
+url = "https://nemlig.com"
```
Comment (tomkerkhove) [inline config.toml:441]: Can you alphabetically sort please?

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
Thanks for opening this PR and using KEDA!

Just one request to fix DCO + Sorting alphabetically.

**Comment by JorTurFer** [comment]
> Can you fix the DCO issues please? You can learn more in our [contribution guide](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md?rgh-link-date=2022-12-09T12%3A32%3A35Z#i-didnt-sign-my-commit-now-what).

it's wait because I can see that the commit is signed, maybe it's something related with the verification?

**Pull request closed without merging**